web/src/i18n/translations/en.yml

```
@@ -139,17 +139,26 @@ Title: ''
+Meta Data: ''
+Version History: ''
 Item Information: ''
 Updated At: ''
 Updated By: ''
 Publish State: ''
-Linked Request: ''
+Customized meta data: ''
+Replace item: ''
+Refer to item: ''
+current: ''
+Created by: ''
```

YAMLlint (1.35.1)

[error] duplication of key "Created by" in mapping

(key-duplicates)